src/client/components/Appheader/AppHeader.component.js

```diff
@@ -1,15 +1,17 @@
 import './AppHeader.style.css';
 
 import React from 'react';
+import { useUser } from '../../context/userContext';
+import { Button } from '../Button/Button';
 
 import Vector from './Vector.svg';
 
 export const AppHeader = () => {
+  const { isAuthenticated } = useUser();
   return (
     <div className="AppHeader-div">
-      <div className="Appheader-img-container">
-        <img src={Vector} alt="AppHeader" />
-      </div>
+      <img src={Vector} alt="AppHeader" className="Appheader-img-container" />
+      {isAuthenticated && <Button className="AppHeader-button">Logout</Button>}
```

Comment:
There is no `className` property on the `Button` component. Do you want to add it?

Also, consider adding a new appearance type called `link` that would style the button as a link so that it is smaller.

Reply:
is `className`  not default from react, when I add a PropTypes as `className` it shows an error that no duplicate props allowed

Reply:
what is this `link`  is it like `<link>` tag

Reply:
`className` is present on built-in components like `div` and `span`, but it is never defined on any components we define.

As for `link`, I said add an _appearance_ called link. Look at the `Button` component and find the `appearance` property. You will notice it has `default` and `danger`. Take inspiration from https://getbootstrap.com/docs/5.0/components/buttons/.